primo/utils/override_utils.py

```diff
@@ -130,43 +134,39 @@ def return_value(self):
         return self.well_list
 
 
-def UserInput(well_df_add: pd.DataFrame, well_df_remove: pd.DataFrame):
+def user_input(well_df_add: pd.DataFrame, well_df_remove: pd.DataFrame):
     """
     A wrapper to generate the override widget for adding wells and removing wells
 
     """
-    widget_add = overridewidget(well_df_add, "Add the well to the suggested projects")
-    widget_remove = overridewidget(
+    widget_add = OverrideWidget(well_df_add, "Add the well to the suggested projects")
+    widget_remove = OverrideWidget(
         well_df_remove, "Remove the well from the suggested projects"
     )
 
     return widget_add, widget_remove
 
 
-class recalculate:
+class Recalculate:
     def __init__(
         self,
         original_well_list: pd.DataFrame,
         well_add_list: list,
         well_remove_list: list,
         well_df: pd.DataFrame,
-        mobilization_costs,
-        budget,
-        max_wells_per_owner,
-        max_distance,
-        dac_weight=None,
-        dac_budget_fraction=None,
+        opt_inputs: OptInputs,
+        dac_weight: float,
```

Comment:
It is possible that under some scenarios DAC is not considered, for example, the Example demo notebook. Should we keep the default value as `None` for the `dac_weight`?